internal/nodes/avi_model_nodes.go

```diff
@@ -27,6 +27,7 @@ import (
 	"github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/internal/lib"
 	"github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pkg/utils"
 
+	"github.com/vmware/alb-sdk/go/models"
```

Comment:
Looks like we have imported the same package twice and even using it with different names in the following code.

Reply:
Changed